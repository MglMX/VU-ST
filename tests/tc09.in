1
a
e
o
t
r
w
