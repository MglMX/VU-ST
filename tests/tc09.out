Choose category to play or type exit to leave
(1) Cars
(2) Animals
Type category number:==============
Category: Cars
Letters: []
Lives left: 6
Word to guess: _ _ _ _ _ _
You missed!
==============
Category: Cars
Letters: ['a']
Lives left: 5
Word to guess: _ _ _ _ _ _
You missed!
==============
Category: Cars
Letters: ['a', 'e']
Lives left: 4
Word to guess: _ _ _ _ _ _
You missed!
==============
Category: Cars
Letters: ['a', 'e', 'o']
Lives left: 3
Word to guess: _ _ _ _ _ _
You missed!
==============
Category: Cars
Letters: ['a', 'e', 'o', 't']
Lives left: 2
Word to guess: _ _ _ _ _ _
You missed!
==============
Category: Cars
Letters: ['a', 'e', 'o', 'r', 't']
Lives left: 1
Word to guess: _ _ _ _ _ _
You missed!
You lost! Correct answer: suzuki
